README.md

```diff
@@ -49,8 +49,8 @@ v2:
 ### Generating and Building clients
 
 From the root javascript-clients folder:
-* To generate all clients run `npm run generate` -- NX will run through our `client-generator` executor located in `@redhat-cloud-services/build-utils` to execute the `openapi-generator-cli` command to generate your client based off of the specs defined in your client's `openapi-spec.json`.
-* To build clients and generate their dist to be published run `npm run build` -- NX will only build packages when it detects that a change has been made to the client (otherwise it will reference the cache). After a client has been built, our `builder` (located in `packages/build-utils`) will move each client's `dist` into a top-level `dist` for publishing. Use `npx nx run-many --skip-nx-cache -t build --exclude=@redhat-cloud-services/CLIENTNAME-client` if you wish to build all clients regardless of whether or not a change has been made.
+* To generate all clients run `npm run generate` -- NX will run through our `client-generator` executor located in `@redhat-cloud-services/build-utils` to execute the `openapi-generator-cli` command to generate your client based off of the specs defined in your client's `openapi-spec.json`. To generate one client, run `nx run @redhat-cloud-services/${your-client-name}-client:generate`. If you don't have `openapi-generator-cli` installed locally, you will need to run `npm install -g @openapitools/openapi-generator-cli`
```

Comment:
@apinkert small nit - can we change `locally` to `globally`?

Reply:
@florkbr makes sense!